packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/Tasks.js

```diff
@@ -48,30 +47,22 @@ class Tasks extends React.Component {
       // there has to be a better way
       // but works for now
       return (
-        <Box as='form' gap='small' justify='between' fill>
-          {step.tasks.map((task) => {
-            // classifications.addAnnotation(task, value) retrieves any existing task annotation from the store
-            // or creates a new one if one doesn't exist.
-            // The name is a bit confusing.
-            const annotation = classification.addAnnotation(task)
-            task.setAnnotation(annotation)
-            const TaskComponent = observer(taskRegistry.get(task.type).TaskComponent)
-            if (annotation && TaskComponent) {
-              return (
-                <Box key={annotation.id} basis='auto'>
-                  <TaskComponent
-                    {...this.props}
-                    disabled={!ready}
-                    annotation={annotation}
-                    task={task}
-                    theme={theme}
-                  />
-                </Box>
-              )
-            }
-
-            return (<Paragraph>Task component could not be rendered.</Paragraph>)
-          })}
+        <Box
+          key={classification.id}
+          as='form'
+          gap='small'
+          justify='between'
+          fill
+        >
+          {step.tasks.map((task) => (
+            <Task
+              key={task.taskKey}
+              {...this.props}
```

Comment:
Should I remove this. I don't think all the task area props need to be passed down to every task, but I'm not 100% sure which props the PH-TESS task needs.

Reply:
I agree. I think the data-vis task only needs a `task` prop, but I'm not sure.

Reply:
The downside of this, I think, is that `useStores`, in the Task component, will run every time one of these extra props changes. That might be why I've been seeing bugs related to the `classification.addAnnotation(task)` action running when it isn't needed.